fusedrug_examples/interaction/drug_target/affinity_prediction/PLM_DTI/losses.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import torch.nn.functional as F
+
+## TODO: move to fuse core
+
+
+def focal_loss(
```

Comment:
is it the same focal_loss as in bmfm?

Reply:
it's a different implementation. this one supports both logit and probability inputs, and one hot labels. I also had some problems with the one in bmfm but didn't do a full comparison to understand the root cause